nidmfsl/fsl_exporter/fsl_exporter.py

```diff
@@ -428,29 +429,129 @@ def _find_inferences(self):
                 zFileImg = filename
 
                 # Cluster Labels Map
-                if self.fsl_path is not None:
-                    cmd = os.path.join(self.fsl_path, "bin", "cluster")
-                    cluster_labels_map = os.path.join(
-                        analysis_dir, 'tmp_clustmap' + stat_num_t + '.nii.gz')
-                    cmd = cmd + " -i " + zFileImg + \
-                                " -o " + cluster_labels_map + " -t 0.01"
-
-                    # Discard stdout
-                    FNULL = open(os.devnull, 'w')
-                    subprocess.check_call(
-                        "cd "+analysis_dir+";"+cmd, shell=True,
-                        stdout=FNULL, stderr=subprocess.STDOUT)
+                cluster_labels_map = os.path.join(
+                    analysis_dir, 'tmp_clustmap' + stat_num_t + '.nii.gz')
 
-                    temporary = True
-                    clust_map = ClusterLabelsMap(
-                        cluster_labels_map, self.coord_space,
-                        suffix=stat_num_t,
-                        temporary=temporary)
+                excset_img = nib.load(filename)
+                # Get cluster connectivity
+                # There is not table display listing peaks and clusters for
+                # voxelwise correction
+                feat_post_log_file = os.path.join(
+                    analysis_dir, 'logs', 'feat4_post')
+                if os.path.isfile(feat_post_log_file):
+                    with open(feat_post_log_file, 'r') as log:
+                        feat_post_log = log.read()
+                    connectivity = 26  # FSL's default
```

Comment:
Is this connectivity hard coded whenever the log is present?

Reply:
Good catch!

Reply:
Solved in 339baaa